pycoal/environment.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+
+import coal
+
+class EnvironmentalCorrelation:
+
+    def correlateMiningEnvironment(environmentFilename, miningFilename, outputFilename):
```

Comment:
If possible, then please implement documentation so we have a better opportunity to discuss this class on Friday. If possible, please do this for all functions.